src/Entities/Nodes.jl

```diff
@@ -89,28 +89,15 @@ function Base.show(io::IO, ::MIME"text/plain", n::Node)
     println("• Node at $(n.x) and dofs $(n.dofs)")
 end
 
-"1D node constructor with one dim coordinates."
-function Node(x₁::T, dofs::Dictionary=Dictionary{Field,Vector{Dof}}()) where {T<:Real}
-    Node(Point(x₁), dofs)
-end
-
-"2D node constructor with two coordinates."
-function Node(x₁::T, x₂::T, dofs::Dictionary=Dictionary{Field,Vector{Dof}}()) where {T<:Real}
-    Node(Point(x₁, x₂), dofs)
-end
-
-"3D node constructor with coordinates."
-function Node(x₁::T, x₂::T, x₃::T,
-              dofs::Dictionary=Dictionary{Field,Vector{Dof}}()) where {T<:Real}
-    Node(Point(x₁, x₂, x₃), dofs)
-end
-
 "Node constructor with a `NTuple`."
 function Node(t::NTuple{dim,T},
               dofs::Dictionary=Dictionary{Field,Vector{Dof}}()) where {dim,T<:Real}
     Node(Point(t), dofs)
 end
 
+"`Node` constructor to promote types of coordinates of Point"
+Node(t, dofs::Dictionary=Dictionary{Field,Vector{Dof}}()) = Node(promote(t...), dofs)
```

Comment:
Cool so we fixed the previous, now seems is failing because in `t` we have also the `Dictionary` and Julia is using the more general method instead of the internal consttructore. Ref: https://discourse.julialang.org/t/understanding-multiple-dispatch/76601. So I suggest specializing to:
```suggestion
Node(t::Union{Tuple, AbstractArray}, dofs::Dictionary=Dictionary{Field, Vector{Dof}}()) = Node(t, dofs)
```